packages/build-utils/test/integration.test.ts

```diff
@@ -32,11 +31,6 @@ const skipFixtures: string[] = [
   '08-zero-config-middleman',
```

Comment:
I think we could remove `01-zero-config-api` but `02-zero-config-api` was actually copied to the new package since it was a test fixture used by two different tests.

Reply:
Ah, I see that now. Yeah, I think we should remove just `01-zero-config-api`, then.